eng/pipelines/templates/jobs/ci.yml

```diff
@@ -253,15 +289,12 @@ jobs:
         - pwsh: |
             python -m pip install "./tools/azure-sdk-tools[build]"
           displayName: 'Prep Environment'
-        - template: /eng/pipelines/templates/steps/targeting-string-resolve.yml
-          parameters:
-            BuildTargetingString: ${{ parameters.BuildTargetingString }}
         - task: PythonScript@0
           displayName: 'Ensure service coverage'
           inputs:
             scriptPath: '$(Build.SourcesDirectory)/scripts/devops_tasks/update_regression_services.py'
             arguments: >-
-              "$(TargetingString)"
+              "azure*"
```

Comment:
Did you intend to hardcode this targeting string now?

Reply:
Yep. The targeting string is actually pointless given that we scope to the service directory. Regression runs as part of nightly, so we will always have a scoping service directory to take care of us.